airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -0,0 +1,85 @@
+package io.airbyte.integrations.destination.mysql;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.Databases;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.destination.jdbc.AbstractJdbcDestination;
+import io.airbyte.protocol.models.AirbyteConnectionStatus;
+import io.airbyte.protocol.models.AirbyteConnectionStatus.Status;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MySQLDestination extends AbstractJdbcDestination implements Destination {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MySQLDestination.class);
+
+  public static final String DRIVER_CLASS = "com.mysql.cj.jdbc.Driver";
+
+  @Override
+  public AirbyteConnectionStatus check(JsonNode config) {
+    try (final JdbcDatabase database = getDatabase(config)) {
+      String outputSchema = namingResolver.getIdentifier(config.get("database").asText());
```

Comment:
Why is the output schema the same as the database?

Reply:
MySQL doesn't have the schema concept. There are only databases.

Reply:
@jrhizor in mysql database and schema are kind of same thing. Unlike postgres where you have a database in which you have schemas and each schema contains tables, in mysql the tables live inside the database only.